app/src/main/java/com/hedvig/app/feature/referrals/ui/editcode/ReferralsEditCodeActivity.kt

```diff
@@ -72,7 +75,34 @@ class ReferralsEditCodeActivity : BaseActivity(R.layout.activity_referrals_edit_
                 }
             }
         }
-        code.setText(currentCode)
+        code.setOnEditorActionListener { _, actionId, _ ->
+            if (actionId == EditorInfo.IME_ACTION_DONE) {
+                code.dismissKeyboard()
```

Comment:
If submitting using IME dismisses the keyboard, then shouldn't the `Save`-button do the same?